modules/storage-persistent-storage-selinuxChangePolicy-pod.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//* microshift_storage/understanding-persistent-storage-microshift.adoc
+
+
+[id="using_selinuxChangePolicy_pod_{context}"]
+= Changing seLinuxChangePolicy at the pod level
+
+You can set the set the `seLinuxChangePolicy` parameter in a new or existing deployment, and then the pods that it manages will have this parameter value. You can similarly do this for a statefulset. You cannot edit an existing pod to set `seLinuxChangePolicy`; however, you can set this parameter when creating a new pod.
+
+This procedure describes how to set the `seLinuxChangePolicy` parameter in an existing deployment.
+
+.Prerequisites
+
+* Access to the {product-title} console.
+
+.Procedure
+
+To set the `seLinuxChangePolicy` parameter in an existing deployment:
+
+. Click *Workloads* > *Deployments*.
+
+. On the *Deployment* page, click the desired deployment.
+
+. On the *Deployment details* page, click the *YAML* tab.
+
+. Edit the deployment's YAML file under `spec.template.spec.securityContext` as per the following example file:
++
+.Example deployment YAML file setting `seLinuxChangePolicy`
+[source,yaml]
+----
+  ...
+securityContext:
+  seLinuxChangePolicy: MountOption <1>
```

Comment:
MountOption is tech preview for RWO/RWX, need to enable feature gate
This is GA with RWOP and I believe default is MountOption
@jsafrane

Reply:
@jsafrane can you confirm that MountOption is the default for RWOP for 4.20?

Reply:
Yes, `MountOption` requires TechPreviewNoUpgrade to be enabled.

Reply:
Fixed.